### ISSUE TITLE
docs-bug(MatFormFieldControl): The guide is missing detailed description of autofilled property

### ISSUE THREAD

**Issue by DmitryEfimenko** [angular/components#26301]
### Documentation Feedback

The guide ["Creating a custom form field control"](https://v10.material.angular.io/guide/creating-a-custom-form-field-control) is missing detailed description of `autofilled` property

### Affected documentation page

https://v10.material.angular.io/guide/creating-a-custom-form-field-control

**Comment by DrVickie8** [comment]
Hello, can you elaborate on this? I looked through the document, I can't find the 'autofilled' property

**Comment by DmitryEfimenko** [comment]
Right. That's the problem. The description is missing. I updated original issue with the [link to the property](https://github.com/angular/components/blob/2f9a59a24c0464cbed7f54cbeb5cba73e6007715/src/material/form-field/form-field-control.ts#L63)

**Comment by amysorto** [comment]
The up to date API docs is here: https://material.angular.io/components/form-field/api#MatFormFieldControl. Although this guide should be updated. I'll move this issue to our docs site repo to track this issue.
